src/app/app.module.ts

```diff
@@ -7,19 +7,16 @@ import { HomeComponent } from './home/home.component';
 import { ReviewsComponent } from './reviews/reviews.component';
 import { MoviesComponent } from './movies/movies.component';
 import { HttpClientModule, HTTP_INTERCEPTORS } from '@angular/common/http';
-import { NewMovieComponent } from './new-movie/new-movie.component';
 import { EditMovieDetailComponent } from './edit-movie-detail/edit-movie-detail.component';
 import { ReactiveFormsModule } from '@angular/forms';
 import { NgbModule } from '@ng-bootstrap/ng-bootstrap';
 import { NewMovieModalComponent } from './new-movie-modal/new-movie-modal.component';
-import { MoviesListComponent } from './movies-list/movies-list.component';
 import { LoginComponent } from './login/login.component';
 import { SignupComponent } from './signup/signup.component';
-import { LogoutComponent } from './logout/logout.component';
 import { AuthInterceptor } from './auth.interceptor';
 
 export const httpInterceptorProviders = [
-  { provide: HTTP_INTERCEPTORS, useClass: AuthInterceptor },
+  { provide: HTTP_INTERCEPTORS, useClass: AuthInterceptor, multi: true },
```

Comment:
What is the purpose of "multi: true" ?

Reply:
The `multi: true` property allows the application to provision multiple instances of this injectable. Most importantly, it is done here because `HTTP_INTERCEPTORS` require `multi` to be `true`. If that doesn't make sense, it's fine because it really is not that important to remember/know what it does. Might be easier to just think of it as an Angular thing we just have to do every time we are providing an HTTP interceptor.